api/src/services/generate-xform.js

```diff
@@ -45,25 +45,30 @@ const transform = (formXml, stylesheet) => {
   });
 };
 
+const replaceLast = (haystack, needle) => {
```

Comment:
This function isn't really a "replace" - maybe rename to "removeLast"?

Reply:
Done.